index.html

```diff
@@ -232,17 +232,17 @@ <h1>Get Started</h1>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
 
-      <form action="/">
+      <form action="https://formspree.io/f/mqkovwwj" method="POST">
         <div class="form">
           <label for="name">Your name</label><br>
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching in your work. Please have a look at the below comparison table to understand the problem in a better way through visualization. Please use colour `#7D7C84` instead of `#000000` and use **10px** font-size instead of **12px**.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 06-37-17](https://github.com/donice/Portfolio-Microverse/assets/66476329/e37ba188-d64a-4320-814a-f1946a3755d9)

</td>

<td>

![Screenshot from 2023-06-08 06-37-05](https://github.com/donice/Portfolio-Microverse/assets/66476329/3419cf09-51fd-4eec-ab61-ad585a51707f)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)